Donation-platform/src/main/java/com/google/model/Charity.java

```diff
@@ -4,6 +4,7 @@
 import java.util.Objects;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Comparator;
```

Comment:
Change (1) to use Comparable: public final class Charity implements Comparable<Charity> {